### PR TITLE
Prepare repo for May 2017 release

### PR DESCRIPTION
**Pull request by mfunk**
Merge DEV into MASTER in preparation for May 2017 release.

## Description
Too many to list here, but basically will address all issues in the current milestone work.

## Related Issue
https://github.com/Esri/solutions-geoprocessing-toolbox/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Version+-+Next%22

## Motivation and Context
Regular release schedule has been a problem with recent changes in team and processes. Trying to get back on track and get tools updated for Incident Analysis release.

## How Has This Been Tested?
Manual and automated tests are in place but failing.

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by mfunk** [comment]
Hi @csmoore I was hoping to merge this PR this week, but will move that to next week. Will you have time to review this Monday or a Tuesday next week?

**Comment by csmoore** [comment]
@mfunk - sorry about the delay - with the number of changes, the task was a bit daunting - but I will at least run the tests before Tues/tomorrow

**Comment by csmoore** [comment]
I opened on 10.4.1 and spot checked and there were a few broken tool links:

image

image

image

File: README.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-###Repository Owner: [Matt](https://github.com/mfunk)
+### Repository Owner: [Matt](https://github.com/mfunk)
 
 * Merge Pull Requests
 * Creates Releases and Tags
 * Manages Milestones
 * Manages and Assigns Issues
 
-###Secondary: [Chris](https://github.com/csmoore)
+### Secondary: [Chris](https://github.com/csmoore)
```
Comment (csmoore) [inline README.md]: Just so I am not a bottleneck next review time, you may want to switch this to someone on the template team

File: suitability/test/test_maot/.project [hunk 2]
```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (csmoore) [inline suitability/test/test_maot/.project]: this and below are some kind of Python UI file and don't need to be in the repo (can be added to .gitignore .project , .pydevproject  )

**Comment by csmoore** [comment]
Running the test suite,

```
INFO: 2017-05-23 09:56:08,747 Running tool (Import WMO Station Data)
ERROR: 2017-05-23 09:56:11,392 Executing: ImportWMOStationData C:\Users\Chris\Documents\GitHub\solutions-geoprocessing-toolbox\utils\test\suitability_tests\data\WMO.gdb WMOStationData C:\Users\Chris\Documents\GitHub\solutions-geoprocessing-toolbox\utils\test\suitability_tests\data\WMOStationFiles C:\Users\Chris\Documents\GitHub\solutions-geoprocessing-toolbox\utils\test\suitability_tests\data\WMO.gdb\WMOFC_output
Start Time: Tue May 23 09:56:10 2017
Failed to execute. Parameters are not valid.
ERROR 000310: The Field name must not start with a number
Failed to execute (ImportWMOStationData).
Failed at Tue May 23 09:56:11 2017 (Elapsed Time: 1.00 seconds)
```

And the nothing logged to the log file for 30 mins (during the suitability: `solutions-geoprocessing-toolbox\utils\test\suitability_tests\` tests)
```
INFO: 2017-05-23 09:56:29,864 Running tool (Import CRU CL2.0 To Raster)
INFO: 2017-05-23 10:30:41,563 Running tool (Subset Raster Workspace)
```

**Pull request merged**